Updating Chapter 8 with more inclusive terms

rmhyman opened the pull request:
The terms master/slave should not be propagated for future technologists to absorb. Made changes to text and images (some resizing may be necessary). I'm not sure this terminology is being used in subsequent chapters, but I'm more than happy to update them as I encounter them.

rust-highfive commented:
Thanks for the pull request, and welcome! The Rust team is excited to review your changes, and you should hear from @andre-richter (or someone else) soon.

Please see [the contribution instructions](https://rustc-dev-guide.rust-lang.org/contributing.html) for more information.

adamgreig commented on microbit/src/08-i2c/README.md:
```diff
@@ -18,15 +18,15 @@ uses two lines to exchange data: a data line (SDA) and a clock line (SCL). Becau
-drives the communication with a slave device. Several devices, both masters and slaves, can be
-connected to the same bus at the same time. A master device can communicate with a specific slave
+This protocol uses a *controller* *target* model where the controller is the device that *starts* and
+drives the communication with a target device. Several devices, both conntrollers and targets, can be
```
```suggestion
drives the communication with a target device. Several devices, both controllers and targets, can be
```

adamgreig reviewed (changes requested):
Thanks! These are also the official terms for I2C now; the spec was updated last year.

I noticed one typo, and also I think it would be better if we could use the SVG version of the new images from Wikipedia - could you try changing to those please?

rmhyman commented:
> Thanks! These are also the official terms for I2C now; the spec was updated last year.
> 
> I noticed one typo, and also I think it would be better if we could use the SVG version of the new images from Wikipedia - could you try changing to those please?

That's good to know that the spec was updated 😃 . I've fixed the typo and updated the images to use the SVG file instead of the PNG version.

adamgreig reviewed (approved):
Thanks!

bors r+
